dns/resolver.py

```diff
@@ -1079,6 +1079,27 @@ def set_flags(self, flags):
 
         self.flags = flags
 
+    @property
+    def nameservers(self):
+        return self._nameservers
```

Comment:
`self._nameservers` should be set to empty list in `Resolver.__init__()`

Reply:
Done